core/src/test/java/com/linecorp/armeria/server/ServerBuilderTest.java

```diff
@@ -64,4 +67,106 @@ public void numMaxConnections() {
         final ServerBuilder sb = new ServerBuilder();
         assertThat(sb.maxNumConnections()).isEqualTo(Integer.MAX_VALUE);
     }
+
+    /**
+     * Makes sure each virtual host can have its custom logger name.
+     */
+    @Test
+    public void setAccessLoggerTest1() {
+        final Server sb = new ServerBuilder()
+                .http(0)
```

Comment:
IIRC. this is redundant. Armeria automatically adds a port if a user did not specify a port.